### PR TITLE
[SDK] Publish flow updates

### PR DESCRIPTION
**Pull request by kumaryash90**

### CODE REVIEW

File: packages/sdk/src/evm/core/classes/contract-deployer.ts [hunk 1]
```diff
@@ -966,7 +1086,8 @@ export class ContractDeployer extends RPCConnectionHandler {
       extendedMetadata &&
       extendedMetadata.factoryDeploymentData &&
       (extendedMetadata.isDeployableViaProxy ||
-        extendedMetadata.isDeployableViaFactory) &&
+        extendedMetadata.isDeployableViaFactory ||
+        extendedMetadata.deployType === "customFactory") &&
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/core/classes/contract-deployer.ts:1090]: this is wrong no? should be autoFactory

File: packages/sdk/src/evm/types/deploy/deploy-options.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { DeployedContractType } from "../any-evm/deploy-data";
 
 export type DeployOptions = {
   forceDirectDeploy?: boolean;
+  customFactory?: string;
```
Comment (joaquim-verges) [inline packages/sdk/src/evm/types/deploy/deploy-options.ts:9]: what does this do?

Comment (kumaryash90) [reply packages/sdk/src/evm/types/deploy/deploy-options.ts]: not using it - removed

**Pull request merged**